src/app/ApplicationTemplate.Client/ApiEndpoints/Authentication/AuthenticationToken.cs

```diff
@@ -4,9 +4,9 @@
 
 namespace ApplicationTemplate.Client;
 
-public record AuthenticationToken
+public class AuthenticationToken
```

Comment:
why this needs to be a class instead of a record?

Reply:
I believe it's because of limitations of the System.Text.Json code-gen-based deserializers.